src/js/views/CovidDashboardView.js

```diff
@@ -0,0 +1,881 @@
+/* eslint-disable no-else-return */
+/* eslint-disable no-alert */
+/* eslint-disable no-unused-vars */
+/* eslint-disable no-underscore-dangle */
+/* eslint-disable vars-on-top */
+/* eslint-disable no-var */
+/* eslint-disable prefer-template */
+/* eslint-disable func-names */
+/* eslint-disable no-use-before-define */
+/* eslint-disable no-nested-ternary */
+/* eslint-disable quotes */
+/* eslint-disable object-shorthand */
+/* eslint-disable class-methods-use-this */
+/* eslint-disable no-undef */
+/* eslint-disable new-cap */
+import Chart from 'chart.js';
+import Keyboard from 'simple-keyboard';
+import EventEmitter from '../models/EventEmitter';
+import {
+  elementFactory, clearElement, getElement, getElements,
+} from '../helpers/domElementsHelper';
+import cData from '../helpers/countries';
+import properties from '../helpers/properties';
+
+const _ = require('lodash');
+
+export default class CovidDashboardView extends EventEmitter {
+  constructor() {
+    super();
+    this.model = [];
+    this.chartData = [];
+    this.evnts = {};
+    this.isNoData = true;
+
+    this.properties = properties;
+    this.createDataStatus();
```

Comment:
i guess this can be moved to an init() method. which later can be called as app.init() in index.js